package/scripts/preinstall

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Checked for installed CLT
+if [[ -e "/Library/Developer/CommandLineTools/usr/bin/git" ]]; then
+  exit 0
+fi
+
+if /usr/bin/xcode-select --install; then
+  exit 0
+fi
```

Comment:
GUI install like this is good, but we should have a branch to employ the non-GUI install method (see install.sh) if `COMMAND_LINE_INSTALL` is set, so that `installer -pkg Homebrew.pkg` works non-interactively as expected.

Reply:
I'm not sure if we have access to environment variables like that.

Reply:
`COMMAND_LINE_INSTALL` is documented in `man installer`.